test/e2e/step_definitions/send.step.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const { defineSupportCode } = require('cucumber');
+const { waitForElemAndMatchItsText } = require('../support/util.js');
+
+defineSupportCode(({ Then }) => {
+  Then('I should see "{elementName}" element with text matching coverter price', (elementName, callback) => {
+    const regexp = /^~ \d{0,100}(\.\d{0,2}){0,1}$/;
```

Comment:
The regexp could be a bit more restrictive `/^~ \d{1,100}(\.\d{1,2})?$/`. Also `?` is equal to and cleaner than `{0,1}`

Anyway, I really like the you're thinking about the future with making sure it will work even when LSK / USD is 1 / 10^100 😆 👍